modules/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+[id="virt-attaching-vm-to-sriov-network_{context}"]
+= Attaching a virtual machine to an SR-IOV network
+
+You can attach the virtual machine to the SR-IOV network by including the network details in the virtual machine configuration. 
+
+.Procedure
+
+. Include the SR-IOV network details in the `spec.domain.devices.interfaces` and `spec.networks` of the virtual machine configuration:
```

Comment:
I'd say "Create a virtual machine configuration file that is named `vm-sriov.yaml` and includes the SR-IOV network details in the `spec.domain.devices.interfaces` and `spec.networks` sections:"

Reply:
It could be an existing VM that we're editing to attach to a new sriov network so I'm trying to keep that open. I'll add the annotation to the command below.